src/clientdlg.cpp

```diff
@@ -1258,6 +1285,14 @@ void CClientDlg::UpdateDisplay()
         chbChat->setChecked ( true );
         chbChat->blockSignals ( false );
     }
+    if ( !pClient->IsRunning() )
```

Comment:
Should be in `CClientDlg::Disconnect()`?

Reply:
Good catch!
Copy paste is good, turning brain on is better :)